### PR TITLE
Rebase to 3.17

### PR DESCRIPTION
**Pull request by thespad**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/docker-mstream/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

### CODE REVIEW

File: root/etc/s6-overlay/s6-rc.d/svc-mstream/run [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/with-contenv bash
+# shellcheck shell=bash
+
+PORT=$(grep '"port"' /config/config.json | awk -F ':' '{print $2}' | tr -d ',')
```
Comment (nemchik) [inline root/etc/s6-overlay/s6-rc.d/svc-mstream/run:4]: should we use `jq` here?

Comment (thespad) [reply root/etc/s6-overlay/s6-rc.d/svc-mstream/run:4]: Yes, probably.

Comment (thespad) [reply root/etc/s6-overlay/s6-rc.d/svc-mstream/run]: Fixed in c2bd0a4

**Review by nemchik — APPROVED** [review]
lgtm

**Pull request merged**